### ISSUE TITLE
tensorflowjs json model not supported

### ISSUE THREAD

**Issue by gangtao** [lutzroeder/netron#270]
for https://lutzroeder.github.io/netron/  it does not support tensorflowjs json model. 

`Error loading model. Unsupported file content for extension '.json' in 'air-time-model.json'.`

**Comment by gangtao** [comment]
Here is my model file:
`{"modelTopology":{"class_name":"Sequential","config":[{"class_name":"LSTM","config":{"units":5,"activation":"sigmoid","recurrent_activation":"hard_sigmoid","use_bias":true,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_avg","distribution":"normal","seed":null}},"recurrent_initializer":{"class_name":"Orthogonal","config":{"gain":1,"seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"unit_forget_bias":null,"kernel_regularizer":null,"recurrent_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"recurrent_constraint":null,"bias_constraint":null,"dropout":0,"recurrent_dropout":0,"implementation":null,"return_sequences":true,"return_state":false,"go_backwards":false,"stateful":false,"unroll":false,"name":"lstm_LSTM1","trainable":true,"batch_input_shape":[null,12,1],"dtype":"float32"}},{"class_name":"LSTM","config":{"units":5,"activation":"sigmoid","recurrent_activation":"hard_sigmoid","use_bias":true,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_avg","distribution":"normal","seed":null}},"recurrent_initializer":{"class_name":"Orthogonal","config":{"gain":1,"seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"unit_forget_bias":null,"kernel_regularizer":null,"recurrent_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"recurrent_constraint":null,"bias_constraint":null,"dropout":0,"recurrent_dropout":0,"implementation":null,"return_sequences":false,"return_state":false,"go_backwards":false,"stateful":false,"unroll":false,"name":"lstm_LSTM2","trainable":true}},{"class_name":"Dense","config":{"units":1,"activation":"linear","use_bias":true,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_avg","distribution":"normal","seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"kernel_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"bias_constraint":null,"name":"dense_Dense1","trainable":true}}],"keras_version":"tfjs-layers 1.0.4","backend":"tensor_flow.js"},"format":"layers-model","generatedBy":"TensorFlow.js tfjs-layers v1.0.4","convertedBy":null,"weightsManifest":[{"paths":["./air-time-model.weights.bin"],"weights":[{"name":"lstm_LSTM1/kernel","shape":[1,20],"dtype":"float32"},{"name":"lstm_LSTM1/recurrent_kernel","shape":[5,20],"dtype":"float32"},{"name":"lstm_LSTM1/bias","shape":[20],"dtype":"float32"},{"name":"lstm_LSTM2/kernel","shape":[5,20],"dtype":"float32"},{"name":"lstm_LSTM2/recurrent_kernel","shape":[5,20],"dtype":"float32"},{"name":"lstm_LSTM2/bias","shape":[20],"dtype":"float32"},{"name":"dense_Dense1/kernel","shape":[5,1],"dtype":"float32"},{"name":"dense_Dense1/bias","shape":[1],"dtype":"float32"}]}]}`

**Comment by lutzroeder** [comment]
[air-tlme-model.json.zip](https://github.com/lutzroeder/netron/files/3170414/air-tlme-model.json.zip)

**Comment by lutzroeder** [comment]
[air-time-model.json.zip](https://github.com/lutzroeder/netron/files/3170424/air-time-model.json.zip)
